flink-runtime/src/test/java/org/apache/flink/runtime/operators/testutils/MockEnvironment.java

```diff
@@ -57,7 +57,7 @@
 import org.apache.flink.util.UserCodeClassLoader;
 import org.apache.flink.util.concurrent.Executors;
 
-import com.sun.istack.NotNull;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Let's just remove this usage. Such annotation shouldn't be used at all, because for new code we go in with the assumption that nothing is nullable unless it is annotated accordingly.

Reply:
Thanks, you are right. Removed